SwiftWisdomTests/StandardLibrary/Numbers/UnsignedInteger+ExtensionsTests.swift

```diff
@@ -19,4 +19,188 @@ class UnsignedIntegerExtensionTests: XCTestCase {
+        XCTAssert(UInt(135).ip_containsMask(4)) // 0b10000111, 0b111 -> true
+        XCTAssertFalse(UInt(131).ip_containsMask(5)) // 0b10000011, 0b101 -> false
+        
+        XCTAssert(UInt(UINT8_MAX).ip_containsMask(UInt(UINT8_MAX)))
+        XCTAssert(UInt(UINT8_MAX).ip_containsMask(3))
+        XCTAssert(UInt(UINT8_MAX).ip_containsMask(0))
+
+    }
+    
+    func test_ip_containsBitMask() {
+        XCTAssert(UInt(0).ip_containsBitMask(0)) // 0b0, 0b0 -> true
+        XCTAssertFalse(UInt(0).ip_containsBitMask(1)) // 0b0, 0b1 -> false
+        
+        XCTAssert(UInt(1).ip_containsBitMask(1)) // 0b1, 0b1 -> true
+        XCTAssertFalse(UInt(1).ip_containsBitMask(10)) // 0b1, 0b0 -> false
+        
+        XCTAssert(UInt(135).ip_containsBitMask(7)) // 0b10000111, 0b111 -> true
+        XCTAssert(UInt(135).ip_containsBitMask(4)) // 0b10000111, 0b111 -> true
+        XCTAssertFalse(UInt(131).ip_containsMask(5)) // 0b10000011, 0b101 -> false
+        
+        XCTAssert(UInt(UINT8_MAX).ip_containsBitMask(UInt(UINT8_MAX)))
+        XCTAssert(UInt(UINT8_MAX).ip_containsBitMask(3))
+        XCTAssert(UInt(UINT8_MAX).ip_containsBitMask(0))
+    }
+    
+    func test_ip_data() {
+        var number = UInt8(0)
+        var correctData = Data(byte: number)
+        XCTAssertEqual(correctData, UInt8(0).ip_data)
+        XCTAssertNotEqual(correctData, UInt8(1).ip_data)
+        
+        number = UInt8(1)
+        correctData = Data(byte: number)
+        XCTAssertEqual(correctData, UInt8(1).ip_data)
+        XCTAssertNotEqual(correctData, UInt8(0).ip_data)
+        
+        number = UInt8(10)
+        correctData = Data(byte: number)
+        XCTAssertEqual(correctData, UInt8(10).ip_data)
+        XCTAssertNotEqual(correctData, UInt8(20).ip_data)
+        
+        number = UInt8(INT8_MAX)
+        correctData = Data(byte: number)
+        XCTAssertEqual(correctData, UInt8(INT8_MAX).ip_data)
+        XCTAssertNotEqual(correctData, UInt8(0).ip_data)
+    }
+    
+    func test_ip_nsdata() {
+        var number = UInt8(0)
+        var correctNSData = NSData(byte: number)
+        XCTAssert(correctNSData == UInt8(0).ip_nsdata)
+        XCTAssertNotEqual(correctNSData, UInt8(1).ip_nsdata)
+        
+        number = UInt8(1)
+        correctNSData = NSData(byte: number)
+        XCTAssertEqual(correctNSData, UInt8(1).ip_nsdata)
+        XCTAssertNotEqual(correctNSData, UInt8(0).ip_nsdata)
+        
+        number = UInt8(10)
+        correctNSData = NSData(byte: number)
+        XCTAssertEqual(correctNSData, UInt8(10).ip_nsdata)
+        XCTAssertNotEqual(correctNSData, UInt8(20).ip_nsdata)
+        
+        number = UInt8(INT8_MAX)
+        correctNSData = NSData(byte: number)
+        XCTAssertEqual(correctNSData, UInt8(INT8_MAX).ip_nsdata)
+        XCTAssertNotEqual(correctNSData, UInt8(0).ip_nsdata)
+    }
+    
+    func test_ip_maskedIndexes() {
+        XCTAssert(UInt(0).ip_maskedIndexes == [])
```

Comment:
The calls in this method should also get the `XCTAssertEqual` / `XCTAssertNotEqual ` treatment.